spec/order_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'order'
+describe Order do
+    let(:hamburger) {double(name: 'Hamburger', price: 5)}
+    let(:chips) {double(name: 'Chips', price: 2)}
+
+    it {is_expected.to respond_to(:add, :total_price, :print_order, :place_order)}
+
+    it 'starts with empty order list' do
+        expect(subject.order_list).to be_empty
+    end
+
+    it 'add dishes to the order list' do
+        subject.add(hamburger)
+        subject.add(chips, 2)
+        expect(subject.order_list).to include({hamburger => 1}, {chips => 2})
```

Comment:
Same again, you should avoid testing `subject.order_list` because this is _state_ not _behaviour_.